doc/cluster_spec.md

```diff
@@ -24,7 +24,9 @@ Some options in a running cluster specification can be changed to update the des
 | synchronousReplication    | use synchronous replication between the master and its standbys                                                                                                                                                                                                                                                                                                                                                                                                                   | no                        | bool              | false                                                                                                                               |
 | minSynchronousStandbys    | minimum number of required synchronous standbys when synchronous replication is enabled (only set this to a value > 1 when using PostgreSQL >= 9.6)                                                                                                                                                                                                                                                                                                                               | no                        | uint16            | 1                                                                                                                                   |
 | maxSynchronousStandbys    | maximum number of required synchronous standbys when synchronous replication is enabled (only set this to a value > 1 when using PostgreSQL >= 9.6)                                                                                                                                                                                                                                                                                                                               | no                        | uint16            | 1                                                                                                                                   |
+| additionalSyncStandbyNames| additional names for possible standbys not included into stolon cluster, but used as replication destinations                                                                                                                                                                                                                                                                                                                                                                     | mo                        | string            | null                                                                                                                                |                    
```

Comment:
Will do this an []string or you'll have to explicitly define that they should be comma separated and parse/validate them.